src/ui/public/doc_table/components/table_row/details.html

```diff
@@ -1,6 +1,20 @@
 <td colspan="{{ columns.length + 2 }}">
-  <a class="pull-right" ng-href="#/doc/{{indexPattern.id}}/{{row._index}}/{{row._type}}/?id={{row._id | uriescape}}">
-    <small>Link to /{{row._index}}/{{row._type}}/{{row._id | uriescape}}</small></i>
-  </a>
+  <div class="documentTableRow__actions">
```

Comment:
👍  Thanks man. I understand the reasoning... the problem is, the button visual style communicates one thing to a user, and links communicate another. We want users to be able to assume they can right-click and open a new tab based on how this link looks. BTW I'm working on aggregating all of these UI and UX ideas in one place so that they're more transparent and available to the team.